atsign-foundation/atsign.dev-3.0
# feat: Added Git Credentials in Footer and Open issue in Header

**- What I did**
fixes #22 
I created a Link to open an issue on the atsign documentation repository and added details for the last commit in the footer as specified.
`Last updated on <date>`
`By <author> in <short commit id>`  (commit id can link to the commit in GitHub)
**- How I did it**

**- How to verify it**
After running the Documentation site on localhost, you can check the header and the footer of the site to verify the same.
![image](https://user-images.githubusercontent.com/76838551/193883887-0c9588cc-7161-4351-9e14-2f17bbd2bdf8.png)
![image](https://user-images.githubusercontent.com/76838551/193883831-3b2c43ef-895c-409e-8ae7-1d27ddb30d0a.png)

**- Description for the changelog**
Updated the header of the site to add a open issue features and added details of last commit in the footer.
